tests/v2/validation/charts/istio.go

```diff
@@ -71,3 +76,28 @@ func getChartCaseEndpointUntilBodyHas(client *rancher.Client, host, path, bodyPa
 
 	return false, nil
 }
+
+// listIstioDeployments is a private helper function
+// that returns the deployment specs if deployments have "operator.istio.io/version" label
+func listIstioDeployments(steveclient *v1.Client) (deploymentSpecList []*appv1.DeploymentSpec, err error) {
+	deploymentListPreUpgrade, err := steveclient.SteveType(workloads.DeploymentSteveType).List(&types.ListOpts{})
```

Comment:
Seems like I forgot to rename this. Since this private helper is used pre and post-upgrade deployment listing, I think it might be better if it's renamed to something like `deploymentList`.

Reply:
You don't think we should just leave the current name?

Reply:
In my opinion, the `listIstioDeployments` name is good. But the local variable `deploymentListPreUpgrade` is not that accurate since `listIstioDeployments` is used both for pre and post-upgrade purposes. So maybe calling that local variable `deploymentList` could be a more generic name. What do you think?

Reply:
Fixed